app/main.py

```diff
@@ -94,9 +95,21 @@ def region_click(feature):
         return get_camera_positions(feature['properties']['code'])
 
 
-@app.callback([Output('layer_style_button', 'children'), Output('alerts_tile_layer', 'url'),
-               Output('alerts_tile_layer', 'attribution')],
-              Input('layer_style_button', 'n_clicks'))
+@app.callback(Output("markers", "children"), [Input("markers", "click_feature")])
+def marker_click(feature):
+    '''
+    This function detect clicks on camera markers and return popup with its area info for now
```

Comment:
```suggestion
    This function detects clicks on camera markers and returns a popup with its area info for now.
```

Just correction of small typos, sorry for being a bit psychorigid when it comes to such things 😅